### PR TITLE
New package SpaDES with dependent packages

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

**Comment by dorton21** [comment]
Any updates on the ability of this to be accepted?

File: var/spack/repos/builtin/packages/r-circstats/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RCircstats(RPackage):
+    """Circular Statistics, from "Topics in Circular Statistics" (2001)
+
+    ircular Statistics, from "Topics in Circular Statistics" (2001)
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/r-circstats/package.py]: Probably want to remove this line

**Comment by tldahlgren** [comment]
> Any updates on the ability of this to be accepted?

This is a lot of packages in one PR.  Have you considered creating a separate PR for each package?

File: var/spack/repos/builtin/packages/subversion/package.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Subversion(AutotoolsPackage):
     version('1.8.17', sha256='1b2cb9a0ca454035e55b114ee91c6433b9ede6c2893f2fb140939094d33919e4')
     version('1.8.13', sha256='17e8900a877ac9f0d5ef437c20df437fec4eb2c5cb9882609d2277e2312da52c')
 
-    variant('serf', default=True,  description='Serf HTTP client library')
+    variant('serf', default=False,  description='Serf HTTP client library')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/subversion/package.py:28]: Curious.  What is the motivation for this change?  And does it apply in general?
